src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,11 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] tableString = (text.replaceAll("[^-?\\d+]", " ").trim().split(" +"));
```

Comment:
```suggestion
        String[] tableString = (text.replaceAll("[^-?\\d+]", " ").trim().split(" "));
```
this plus is redundant here

Reply:
Богдане, але воно тодi створюэ пустi елементи i менi потрiбно робити перевiрку на це. Я (без stream) нiяк не можу сконвертувати одразу ж у масив int

Reply:
можешь, пожалуйста, еще раз попробовать исправить код так, как советует Богдан?
все должно отрабатывать корректно

если возникнут проблемы, опиши их максимально подробно в приватный чат